### DISCUSSION TITLE
表格表头tips如何配置？

### DISCUSSION THREAD

**Discussion by wangdp96** [antvis/S2#3024] [Q&A]
如何给表格的表头添加tips提示，类似于小icon，鼠标移上去展示tips内容

**Comment by lijinke666** [comment]
heaerActionIcons => onClick/onHover => s2.showTooltip({})

https://s2.antv.antgroup.com/manual/advanced/custom/custom-icon

https://s2.antv.antgroup.com/examples/interaction/custom/#row-col-hover-tooltip
